keyboards/skyloong/gk980/bl/ansi/info.json

```diff
@@ -0,0 +1,232 @@
+{
+    "keyboard_name": "GK980 Backlight ANSI_1.0.0",
```

Comment:
Please keep in mind that this is the USB product string and effectively what the device name will be in the OS (although it is not shown everywhere in Windows).

Reply:
I know. The product name is described this way to make it easier for users to understand the keyboard and connect it correctly in VIA.